packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -77,6 +77,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     super.debugOwner,
     this.dragStartBehavior = DragStartBehavior.start,
     this.velocityTrackerBuilder = _defaultBuilder,
+    this.onlyDispatchDragCallbacksWhenThresholdMet = false,
```

Comment:
Just checking, is this the same default behavior as before?

Reply:
That is correct.